testnet-contracts/scripts/sendApproveTx.js

```diff
@@ -35,7 +35,7 @@ module.exports = async () => {
    *** https://github.com/trufflesuite/truffle/issues/889#issuecomment-522581580
    ******************************************/
   if (NETWORK_ROPSTEN) {
-    if (NUM_ARGS !== 3) {
+    if (NUM_ARGS !== 3 && NUM_ARGS !== 4) {
```

Comment:
is this correct? why not change it to ≠ 4?

Reply:
`NUM_ARGS` was confusing everywhere it was used. I can't remember exactly the different arg scenarios for this specific situation and would rather merge these changes and make a new issue to clarify arg usage across the files.

Reply:
https://github.com/cosmos/peggy/issues/145